ansible/roles/csm.ncn.sysctl/README.md

```diff
@@ -0,0 +1,57 @@
+csm.ncn.sysctl
+=========
+
+Set kernel parameter values with sysctl.
+
+Requirements
+------------
+
+The values must exist in configuration prior to running this role. 
+This role must be run in the context of the CSM Configuration
+Framework Service (CFS) in its Ansible Execution Environment (AEE).
+
+Role Variables
+--------------
+
+Available variables are listed below, along with default values (located in
+`defaults/main.yml` and `vars/main.yml`):
+
+    sysctl_config: 'dict'
```

Comment:
The earlier text claims that this is the default value of this variable, but that doesn't actually seem to be the case, unless I am misunderstanding something (which is entirely possible!)

Reply:
I'm not sure I understand your question. As in, because there's nothing in defaults/main.yml for sysctl_config?

Reply:
Adjusted to not mention default value, since `sysctl_config` is a dictionary of potentially a lot of value and the "default" we provide is really just an example based on a few standard values that are already set.